hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/file/FileSqlConnector.java

```diff
@@ -159,4 +160,17 @@ public Vertex fullScanReader(
     public boolean supportsExpression(@Nonnull HazelcastRexNode expression) {
         return true;
     }
+
+    @Override
+    public Set<String> secureConnectorOptions() {
+        Set<String> set = SqlConnector.super.secureConnectorOptions();
+        // TODO: discuss if OPTION_PATH and OPTION_GLOB are safe to output.
+        set.addAll(Set.of(
+                OPTION_PATH,
```

Comment:
to be on the safe side I would not include path and glob

Reply:
OTOH secrets are usually passed in separate options (https://docs.hazelcast.com/hazelcast/5.3/sql/mapping-to-a-file-system#supported-file-systems), so path and glob should not be that sensitive.
@kwart what is your opinion about that?

Reply:
I wouldn't hide the `path` and `glob` options.